graphics/oidn/Portfile

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        OpenImageDenoise oidn 1.2.0 v
+github.tarball_from releases
+extract.suffix      .src${extract.suffix}
+categories          graphics
+platforms           darwin
+license             Apache-2
+maintainers         @jasonliu-- openmaintainer
+homepage            http://www.openimagedenoise.org/
+description         denoising filters for images rendered with ray \
+                    tracing
+long_description    Open Image Denoise is a library of ${description}, \
+                    developed at Intel. Its purpose is to provide an \
+                    efficient denoising library that allows \
+                    ray-tracing-based rendering applications to \
+                    significantly reduce rendering times.
+
+checksums           rmd160  0048ae6a28a6466d81d8b62ad2382326d4e8fb6f \
+                    sha256  041f59758e79f4ea29a9b7a952f2c096426820678a5a713880b6d8a6519a75d0 \
+                    size    45522650
+
+depends_build-append \
+                    port:ispc
+depends_lib-append  port:tbb \
+                    port:openimageio
+
+compiler.cxx_standard 2011
+compiler.blacklist-append clang
```

Comment:
is it really necessary to blacklist every Xcode clang compiler ?

Reply:
Can you please comment here ? Why do you need to blacklist every Xcode clang ? If older versions need blacklisting, for some reason, please comment on that reason in the port file, and limit the blacklist to only those versions.

Reply:
Sorry about that, I missed seeing these comments until just now.

Yes, I think that Xcode clang has to be blacklisted. OIDN requires a compiler that has OpenMP support in order to compile. I realize that I'm using a pretty ancient Xcode (8.2), but as far as I know, no version of Xcode supports OpenMP. If there are any versions of Xcode clang that do have OpenMP support built-in, please let me know and I can make sure to only blacklist other versions.

I'll put this info into a comment.

Reply:
Right, thanks for the info. That makes sense. @kencu  can probably confirm but I think you are correct in that no Xcode compiler supports OpenMP, in which case this is I think OK. But I'm not really an expert on this.

Reply:
b.t.w. does this mean you need to declare a dependency on `libomp` ?

Reply:
No, I don't believe so. The `-fopenmp-simd` flag was specifically designed to allow using features of OpenMP when compiling the source code, but avoids the need for any sort of dependency on `libomp`.

Reply:
OK, thanks for the info.